issue_template.md

```diff
@@ -0,0 +1,104 @@
+## ownCloud Core
+
+### Steps to reproduce
+1.
+2.
+3.
+
+### Expected behaviour
+Tell us what should happen
+
+### Actual behaviour
+Tell us what happens instead
+
+### Server configuration
+**Operating system**:
+
+**Web server:**
+
+**Database:**
+
+**PHP version:**
+
+**ownCloud version:** (see ownCloud admin page)
+
+**Updated from an older ownCloud or fresh install:**
+
+**List of activated apps:**
+
+```
+If you have access to your command line run e.g.:
+sudo -u www-data php occ app:list
+from within your ownCloud installation folder
+```
+
+**The content of config/config.php:**
+
+```
+If you have access to your command line run e.g.:
+sudo -u www-data php occ config:list system
+from within your ownCloud installation folder
+
+or 
+
+Insert your config.php content here
+(Without the database password, passwordsalt and secret)
+```
+
+**Are you using external storage, if yes which one:** local/smb/sftp/...
```

Comment:
This as well as encryption, external user-backend and LDAP as well seems irrelevant for us, no? Sure we need info, but we should also not make it a crazy chore to report issues.